src/main/frontend/src/components/shoppage/ShopCategoryItem.js

```diff
@@ -0,0 +1,66 @@
+import classes from "./ShopCategoryItem.module.css";
+import { useEffect, useState } from "react";
+import { AirlineSeatFlat } from "@material-ui/icons";
+
+function ShopCategoryItem({
+  category,
+  onItemChange,
+  selected,
+  selectedSuperCategory,
+  onSuperCategoryChange,
+}) {
+  const [isCollapsed, setIsCollapsed] = useState(false);
+  const [categoryFetched, setCategoryFetched] = useState(false);
+
+  useEffect(async () => {
+    if (category) {
+      setCategoryFetched(true);
+      if (selectedSuperCategory === category.categoryId) {
+        setIsCollapsed(true);
+      }
+    }
+  }, [category]);
+
+  const renderOptions = () => {
+    return (
+      <div>
+        {category.subcategories.map((category) => (
+          <div className={classes.subcategoriy_item}>
+            <input
+              disabled
+              className={classes.category_checkbox}
+              type="checkbox"
+              name={category.categoryId}
+              value={category.categoryId}
+              onChange={onItemChange}
+              checked={selected == category.categoryId ? "checked" : ""}
+            />
+            {" " + category.name}
```

Comment:
why " "?

Reply:
To add little bit of space between checkbox and label if I don't add " " then there will be one next to the other and it doesn't look right